modules/thunder_updater/thunder_updater.drush.inc

```diff
@@ -14,13 +16,16 @@ function thunder_updater_drush_command() {
   $items['thunder-updater-generate-update'] = [
     'description' => 'Generate update definition for module configuration.',
     'arguments' => [
-      'module' => 'Module name',
-      'update-name' => 'Update name',
+      'filename' => 'Filename of generated update',
+      'module' => 'List of modules, leave empty for all',
+    ],
+    'options' => [
+      'filter' => 'Regex to filter the module list',
     ],
     'drupal dependencies' => ['thunder_updater'],
-    'required-arguments' => 2,
+    'required-arguments' => 1,
     'examples' => [
-      'drush thunder-updater-generate-update thunder_media thunder_media__update_8099' => 'Generate patch to update configuration from current installed configuration to configuration provided in files.',
+      'drush thunder-updater-generate-update mypatchfile.yml --filter=/thunder/' => 'Generate patch to update configuration from current installed configuration to configuration provided in files.',
```

Comment:
Can you also add 2nd example with usage of `module` argument. Also it would be nice to rename argument to `modules`, fe.